src/components/Landing.js

```diff
@@ -454,7 +454,7 @@ export default class Landing extends Component {
       }).sort(function(a, b) {
         return dateCompare(a[startDateFieldName], b[startDateFieldName]);
       });
-
+      const getRealNumber = o => o && !isNaN(o) ? o : 0;
```

Comment:
Can you add a comment above describing what's being worked around? What non-numeric values are you seeing here?